src/Hamiltonians/FroehlichPolaron.jl

```diff
@@ -0,0 +1,145 @@
+"""
+    FroehlichPolaron(address::OccupationNumberFS{M}; kwargs...) <: AbstractHamiltonian
+
+
+The Froehlich polaron Hamiltonian for a 1D lattice with `M` momentum modes is given by
+
+```math
+H = \\frac{(p̂_f - p)^2}{m} + ωN̂ - v Σₖ(âₖ^† + â_{-k})
```

Comment:
Would it be worth making this even easier to read in the terminal as

```suggestion
H = (p̂_f - p)^2/m + ωN̂ - v Σₖ(âₖ^† + â₋ₖ)
```

This makes the LaTeX a bit uglier, so I'm unsure.

Reply:
No, it's a good idea. I wasn't aware that you could make `â₋ₖ` 😉